packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -107,7 +107,10 @@ describe("deployments", () => {
 			Author:        Jean-Luc-Picard@federation.org
 			Source:        Rollback from Wrangler 🤠
 			Rollback from: MOCK-DEPLOYMENT-ID-2222
-			🟩 Active"
+			🟩 Active
+
+
+			NOTE: Deployment ID is now referred to as Version ID. The output of this command will be changing in a future version of Wrangler to reflect this. To learn more visit: https://developers.cloudflare.com/workers/configuration/versions-and-deployments"
```

Comment:
"... is now referred to ..." feels a bit ambiguous. I ask myself, "Where is it referred to as Version ID?", "Where is it not?", "How do I know which and when?"

Should this messages explicitly answer these questions?
Or perhaps it would be enough to phrase this as:

```suggestion
			NOTE: The term "Deployment ID" in this output will be changed to "Version ID" in a future version of Wrangler. To learn more visit: https://developers.cloudflare.com/workers/configuration/versions-and-deployments"
```

Reply:
cc @tanushreesharma-cf

Reply:
Minor tweak to Pete's suggestion. Removed "The term" since it's not just the term that's changing but the definition. 

```suggestion
			NOTE: "Deployment ID" in this output will be changed to "Version ID" in a future version of Wrangler. To learn more visit: https://developers.cloudflare.com/workers/configuration/versions-and-deployments"
```

Reply:
https://github.com/cloudflare/workers-sdk/pull/5400/commits/64ed9086802df55c13e6862510f2881fbc25ed2a